packages/create-app/templates/default-app/yarn.lock

```diff
@@ -0,0 +1 @@
+# intentionally left empty
```

Comment:
without this, when running create-app locally, yarn3 yells that the newly created app seems to be an unexpected part of a larger repo (yeah it's inside the monorepo). Creating an empty yarn.lock file that gets immediately overwritten was the most simple fix

Reply:
Talked about this offline and decided to keep for now